src/Phake/ClassGenerator/MockClass.php

```diff
@@ -385,6 +385,7 @@ protected function implementMethod(\ReflectionMethod $method, $static = false)
         }
 
         $returnHint = '';
+	$attributes = '';
```

Comment:
Please use *4 spaces* here as indentation.

Reply:
I used the GitHub editor to do this and I thought it put 4 spaces... Fixed by using my IDE instead :smile: